scripts/exportlegends.lua

```diff
@@ -26,12 +26,508 @@ local MAPS = {
     "Diplomacy",
 }
 
+getItemSubTypeName = function (itemType, subType)
```

Comment:
@mason11987 Not that it makes a difference, but is there a reason for using `getItemSubTypeName = function` instead of `function getItemSubTypeName` that I'm not aware of?